<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManager4NewEdit</class>
 <widget class="QDialog" name="ProfileManager4NewEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile Editor Gen 4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #274050;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: black;
	background-color: yellow;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QComboBox {
	background-color: #1D303C;
	border: none;
	border-radius: 6px;
	color: white;
	padding-left: 13px;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QComboBox QListView{
	color: white;
	background-color: #1D303C;
	border:1px solid black;
	padding-left: 3px;
}

QComboBox::item:alternate {
	background: #1D303C;
}

QComboBox::item:selected {
	border: 1px solid transparent;
	background: #003941;
}

QComboBox::indicator{
	background-color:transparent;
	selection-background-color:transparent;
	color:transparent;
	selection-color:transparent;
}

QComboBox::editable {
	background-color: #1D303C;
	border: none;
	border-radius: 6px;
	color: white;
}

QLabel {
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QCheckBox::indicator {
	border: 2px solid white;
	border-radius: 3px;
	height: 10px;
	width: 10px;
}

QCheckBox::indicator:unchecked {
	background-color: #274050;
}

QCheckBox::indicator:checked {
	background-color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelProfile">
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditProfile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelTID">
     <property name="text">
      <string>TID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QTextBox" name="textBoxTID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonAccept">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelVersion">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxVersion">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Diamond</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pearl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Platinum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heart Gold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Soul Silver</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelSID">
     <property name="text">
      <string>SID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QTextBox" name="textBoxSID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxLanguage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ENG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SPA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FRE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ITA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DEU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JPN</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelDualSlot">
     <property name="text">
      <string>Dual Slot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboBoxDualSlot">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ruby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sapphire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leaf Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emerald</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Radio</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="comboBoxRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hoenn Sound</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sinnoh Sound</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditProfile</tabstop>
  <tabstop>comboBoxVersion</tabstop>
  <tabstop>comboBoxLanguage</tabstop>
  <tabstop>textBoxTID</tabstop>
  <tabstop>textBoxSID</tabstop>
  <tabstop>pushButtonAccept</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
